pom.xml

```diff
@@ -21,7 +21,7 @@
     <name>hazelcast-jclouds</name>
     <groupId>com.hazelcast</groupId>
     <artifactId>hazelcast-jclouds</artifactId>
-    <version>3.7.2-SNAPSHOT</version>
+    <version>3.9-SNAPSHOT</version>
```

Comment:
Why not make use of HZ 3.9 final?

Reply:
It's the project version of `hazelcast-jclouds`, not the Hazelcast version.

Reply:
Ok

Reply:
I think this should stay as 3.7.2-SNAPHOST because next release is 3.7.2 then we can make it 3.7.3-SNAPSHOT. cc: @googlielmo

Reply:
I don't know which version schema you prefer, but I thought it would be more obvious for the users to bump it to 3.9, so it's clear to which HZ version this plugin belongs.

Since this is already merged, please fix this via a direct commit, if your release plan requires 3.7.x instead of 3.9.x as next version.